src/v/pandaproxy/schema_registry/store.h

```diff
@@ -390,6 +391,12 @@ class store {
             }
         }
 
+        // Once we have hit the maximum version number, we can't continue on
+        if (maxver == std::numeric_limits<schema_version::type>::max()) {
+            vlog(plog.info, "Exhausted available versions for subject {}", sub);
```

Comment:
The store doesn't usually log, since errors are already logged with more context:
```
INFO  2025-01-23 18:09:47,294 [shard 0:main] pandaproxy - store.h:396 - Exhausted available versions for subject simple-value
WARN  2025-01-23 18:09:47,294 [shard 0:main] pandaproxy - server.cc:132 - Request: POST /subjects/simple-value/versions failed: pandaproxy::schema_registry::exception (Versions exhausted for subject simple-value)
```
You could argue the line number is useful, and I could argue there's only one all site.